lemur/static/app/index.html

```diff
@@ -53,6 +53,7 @@
                     <li><a ui-sref="authorities">Authorities</a></li>
                     <li><a ui-sref="endpoints">Endpoints</a></li>
                     <li><a ui-sref="notifications">Notifications</a></li>
+                    <li><a ui-sref="api_keys">Api Keys</a></li>
```

Comment:
I think the listing of API keys would fall under "Admin" functionality and should be moved to the Admin dropdown.

Reply:
I originally had that, but then I realized there'd be no way for a non-Admin user to view their API Keys. So I extracted it to the root level.

Reply:
I believe all users can view other users? Additionally, I don't believe we hide the Admin dropdown from regular users.

Reply:
Oh if we don't hide the admin dropdown for regular users we can move it there. I'll go ahead, and move it there (then confirm a regular user can see it).